File: sources/第14天/2代码/04-源代码/12-获取硬件信息/12-获取硬件信息/ViewController.m
//
//  ViewController.m
//  12-获取硬件信息
//
//  Created by Romeo on 15/12/19.
//  Copyright © 2015年 heima. All rights reserved.
//

#import "ViewController.h"
#import "SystemServices.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UITextView *textView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    SystemServices * sys =[SystemServices sharedServices];
    
    self.textView.text = [sys.allSystemInformation description];
}



@end

File: Lottery(彩票)/Support Files/Home/ViewController.swift
//
//  ViewController.swift
//  Lottery(彩票)
//
//  Created by mac on 2018/3/14.
//  Copyright © 2018年 modi. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        setupBgWithPull()
//        setupBgWithTile()
        
        //代码添加幸运转盘View
        setupView()
        
    }
    
    private func setupView() {
        let lucky = LuckyView.luckyView()
        self.view.addSubview(lucky)
        //设置居中显示
        lucky.center = self.view.center
        
        //设置转盘旋转
        lucky.startRotate()
    }

    /**
     *  Desc:以拉升方式绘制图片到View上
     *  Param:
     */
    private func setupBgWithPull() {
        let image = UIImage(named: "LuckyBackground")
        //图片不够大不能通过平铺方式添加到背景上而是通过拉升绘制到背景
        self.view.layer.contents = image?.cgImage //内部自动调用系统绘制
    }
    
    /**
     *  Desc:以平铺方式绘制图片到View上
     *  Param:
     */
    private func setupBgWithTile() {
        let image = UIImage(named: "LuckyBackground")
        //图片不够大不能通过平铺方式添加到背景上而是通过拉升绘制到背景
        self.view.backgroundColor = UIColor.init(patternImage: image!)
    }
    
}

